### PR TITLE
Add ground mode sidestick cursor

### PR DESCRIPTION
**Pull request by FarukEroglu2048**
**Summary of Changes**
The mod adds the sidestick cursor displayed in ground mode.

**Screenshots (if necessary)**

**Additional context**

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css [hunk 1]
```diff
@@ -97,6 +97,22 @@ a320-neo-pfd-element {
...
         border: none;
         transform: scale(0.9); }
+      a320-neo-pfd-element #Mainframe #HorizonContainer #GroundCursor {
+        position: absolute;
+        left: 40.9%;
+        top: 47.7%;
+        width: 7%;
+        height: 7%;
+        border: none;
+        display: block; }
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css:107]: Please put the curly brace on the next line, the instances of it being on the same line is from original game code :)

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css [hunk 2]
```diff
@@ -97,6 +97,22 @@ a320-neo-pfd-element {
...
+      a320-neo-pfd-element #Mainframe #HorizonContainer #GroundCursorLimits {
+        position: absolute;
+        left: 25%;
+        top: 35%;
+        width: 37.5%;
+        height: 10%;
+        border: none;
+        display: block; }
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css:115]: Ditto.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js [hunk 3]
```diff
@@ -111,10 +114,27 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
             this.vsFlash.setAttribute("visibility", "hidden");
         }
 
-        var ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
-        var PFDAlignedFirst = SimVar.GetSimVarValue("L:A32NX_ADIRS_PFD_ALIGNED_FIRST", "Bool");
+        let IsOnGround = SimVar.GetSimVarValue("SIM ON GROUND", "Bool");
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js:117]: Can be `const`.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js [hunk 4]
```diff
@@ -111,10 +114,27 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
...
         var PFDAlignedATT = SimVar.GetSimVarValue("L:A32NX_ADIRS_PFD_ALIGNED_ATT", "Bool");
 
+        if (IsOnGround && PFDAlignedATT) {
+            this.groundCursor.style.display = "block";
+            this.groundCursorLimits.style.display = "block";
+        }
+        else {
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js:124]: Please put the `else` on the same line as the closing bracket of the `if` block, like `} else {`.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js [hunk 5]
```diff
@@ -111,10 +114,27 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
...
+            this.groundCursor.style.display = "none";
+            this.groundCursorLimits.style.display = "none";
+        }
+
+        let YokeXPosition = 40.9 + (15 * SimVar.GetSimVarValue("YOKE X POSITION", "Position"));
+        let YokeYPosition = 47.7 - (15 * SimVar.GetSimVarValue("YOKE Y POSITION", "Position"));
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js:130]: Both can be `const`.

File: A32NX/layout.json [hunk 6]
```diff
@@ -1,644 +1,644 @@
 {
```
Comment (Benjozork) [inline A32NX/layout.json:1]: We have a CI job to edit this file, no need to change it manually. Please revert it as it pollutes the diff log ;)

File: A32NX/manifest.json [hunk 7]
```diff
@@ -29,5 +29,5 @@
             "OlderHistory": ""
         }
     },
-    "total_package_size": "00000000000094035750"
-}
+    "total_package_size": "00000000000094298442"
```
Comment (Benjozork) [inline A32NX/manifest.json:32]: Ditto.

**Review by Benjozork — COMMENTED** [review]
Hello, thank you for your PR !

Do you have any references to demonstrate this behaviour in the real aircraft ? Please provide them so we can verify your changes and test them against IRL A320s !

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css (hunk 1 above)
Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css:107]: its fine to stay consistent with the existing code

**Comment by AdenFlorian** [comment]
thanks for your changes! I made some tweaks, but I think this still needs more work from comparing it to IRL pictures. I've created an issue to track improvement on this, and will merge this now as it is still an improvement.

**Pull request merged**